### PR TITLE
Match benefits from other provinces

### PR DESCRIPTION
**Pull request by DaceyTom2**
# Description 📝

Give a few lines on what you did with this PR!

## Taiga Link 🔗

[ your tiaga issue link here ]

## Checklist ✅
- [ ] created story for all visual components
- [ ] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

File: src/i18n/en.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ export const en = {
   matchMeToBenefits: "Match me to Benefits",
   allEligibleBenefitsHeader: "All Benefits",
   eligibleBenefitsHeader: "Benefits that apply to you",
```
Comment (Moro-Code) [inline src/i18n/en.js]: I would just change this to Service Canada benefits that apply to you

**Review by Moro-Code — APPROVED** [review]
Looks good, when you go back to the home page it resets because of the triedElegibility flag but for a demo its fine

Edit: Saw selector

File: src/redux/selectors/benefits/benefitsDataSelectors.js [hunk 2]
```diff
@@ -72,14 +74,43 @@ export const eligibleBenefitsSelector = createSelector(
...
+);
+
+export const externalBenefitsDataSelector = createSelector(
+  languageSelector,
+  benefitsMapSelector,
+  benefitsMapSelectorFr,
+  benefitsEligibilitySelector,
+  (lang, benefitsData, benefitsDataFr, eligibleBenefits) => {
+    if (!eligibleBenefits) return [];
+    var externalBenefits = [];
```
Comment (Moro-Code) [inline src/redux/selectors/benefits/benefitsDataSelectors.js:101]: lots of unnecessary code, can be solved by using a map function, see other selectors

**Review by Moro-Code — CHANGES_REQUESTED** [review]

File: src/redux/selectors/benefits/benefitsDataSelectors.js (hunk 2 above)
Comment (Moro-Code) [inline src/redux/selectors/benefits/benefitsDataSelectors.js]: change to const or let

File: src/redux/selectors/benefits/benefitsDataSelectors.js [hunk 3]
```diff
@@ -24,6 +24,8 @@ export const benefitTransformer = (data, lang) => {
         : `select ${data["title"]}`,
     isSelected: data.isSelected,
     isEligible: data.isEligible,
+    serviceType: data["service_type"],
+    benefitTag: data["service_type"],
```
Comment (var-kyle) [inline src/redux/selectors/benefits/benefitsDataSelectors.js:28]: benefitTag is getting the wrong data, shout be "benefit_tag"

**Review by var-kyle — CHANGES_REQUESTED** [review]

File: src/redux/selectors/benefits/benefitsDataSelectors.js (hunk 3 above)
Comment (Moro-Code) [reply src/redux/selectors/benefits/benefitsDataSelectors.js:28]: We're using the tag to distinguish if its an internal or external benefit. This is correct I think

Comment (var-kyle) [reply src/redux/selectors/benefits/benefitsDataSelectors.js:28]: If that is the case, what is the purpose of serviceType?

File: src/i18n/en.js (hunk 1 above)
Comment (DaceyTom2) [reply src/i18n/en.js]: Done

**Review by var-kyle — APPROVED** [review]

File: src/redux/selectors/benefits/benefitsDataSelectors.js (hunk 2 above)
Comment (DaceyTom2) [reply src/redux/selectors/benefits/benefitsDataSelectors.js]: var is now let

File: src/redux/selectors/benefits/benefitsDataSelectors.js (hunk 3 above)
Comment (DaceyTom2) [reply src/redux/selectors/benefits/benefitsDataSelectors.js:28]: It is being used to determine which grid the benefit should appear. External vs Internal... Not the best implementation but find for prototype/demo

**Pull request merged**